collector/pg_extensions.go

```diff
@@ -0,0 +1,112 @@
+package collector
+
+import (
+	"context"
+	"database/sql"
+	"github.com/go-kit/log"
+	"github.com/prometheus/client_golang/prometheus"
+	"strconv"
+)
+
+func init() {
+	registerCollector("extensions", defaultEnabled, NewExtensionsCollector)
+}
+
+var pgExtensions = map[string]*prometheus.Desc{
+	"pg_available_extensions": prometheus.NewDesc(
+		"pg_available_extensions",
+		"Extensions that are available for installation",
+		[]string{
+			"name",
+			"default_version",
+			"installed_version",
+		},
+		prometheus.Labels{},
+	),
+	"pg_extensions": prometheus.NewDesc(
+		"pg_extensions",
+		"Installed extensions",
+		[]string{
+			"extname",
```

Comment:
why ext prefixes?

Reply:
probably it means 'extension'

removed it